source/documentation/additional-information.html.md

```diff
@@ -6,50 +6,59 @@ description: Software developers, designers, product owners or business analysts
 
 # Additional Information
 
-If MTD customers are earning any type of income in addition to their self-employment and property income, they must report that to HMRC just as they would in their Self Assessment return. 
+If Making Tax Digital (MTD) customers are earning any type of income in addition to their self-employment and property income, they must report that to HMRC just as they would in their Self Assessment return. 
 
-The software should prompt customers to provide details about any additional income sources. If the software does not support submission of any applicable additional income, customers must use [HMRC online services](/guides/income-tax-mtd-end-to-end-service-guide/documentation/online-tax-account.html#hmrc-online-services) to make their submission. For more information about which type of income to tell HMRC, refer to [Check if you need to tell HMRC about additional income (GOV.UK)](https://www.gov.uk/check-additional-income-tax). 
+>Currently, the endpoints for reporting additional income through software are included in the Individuals Income Received API. As of June 2024, the Individuals Income Received API will be split into the following APIs:<ul><li>Individuals Employments Income API</li> 
```

Comment:
Why say 'June 2024'? The API has been split already and the 8 new APIs have already been deployed to PROD.

Also, if making a 'forward looking' statement, I suggest writing 'From June 2024,...'

Should we also say that the Individuals Income Received API will be deprecated/removed at some point?

Reply:
This was what was agreed with Nikhila - see https://jira.tools.tax.service.gov.uk/browse/MTDSA-23012

Reply:
OK. It looks as if this issue has further to go. Will wave through for now.